frontend/src/Types/Friend.ts

```diff
@@ -0,0 +1,11 @@
+export type Friend = {
+    id: string;
+    name: string;
+    firstName: string;
+    lastName: string;
+    email: string;
+    phone?: number;
+    bio?: string;
+    picture: string;
```

Comment:
Does every user have a picture´?

Reply:
Yes, I always get a picture from the google api. Either it's the picture, that the user uploaded, or its just a placeholder picture with the first letter of the name of the user